templates/python/setup.cfg

```diff
@@ -1,8 +1,10 @@
 [flake8]
-max-complexity = 10
+extend-ignore = E203
```

Comment:
could you add a comment why those two settings are necessary, i.e. these are the defaults that make it work compatible with Black's formatting
perhaps also include a link to the docs

Reply:
done